src/proto/tactic.proto

```diff
@@ -178,3 +119,38 @@ message ShadowEnemyTactic
 }
 
 message StopTactic {}
+
+enum ExcessiveDribblingMode
+{
+    // Allow excessive dribbling (i.e. more than 1 meter at a time)
+    ALLOWED = 0;
+
+    // Lose ball control if we excessively dribble
+    NOT_ALLOWED = 1;
```

Comment:
`LOSE_BALL` is probably better, it's a bit confusing when you had to compare it with `TERMINATE`